docs/tutorials/intro/initializing_brigade.ipynb

```diff
@@ -0,0 +1,232 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# ignore this cell, this is just a helper cell to provide the magic %highlight_file\n",
+    "%run ../../../examples/highlighter.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initializing Brigade\n",
+    "\n",
+    "Easiest way of initializing brigade is with the function [InitBrigade](../../ref/api/brigade.rst#initbrigade).\n",
+    "\n",
+    "With `InitBrigade` you can initialize brigade with a configuration file, with code or with a combination of both.\n",
+    "\n",
+    "Let's start with [a configuration file](../../ref/configuration/index.rst):"
```

Comment:
I think it would be good to mention the key `SimpleInventory` within the config file and where we find the arguments to that class. Even though it's somewhat self explanatory there's still a link to the config object which doesn't mention this setting.